react-frontend/src/core-components/Button/ButtonComponent.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+const ButtonComponent = ({ className, type, onClick, size, text, icon }) => (
```

Comment:
Unless specific customizations are required, components imported from "react-bootstrap" can be directly re-exported.

e.g.
```
import { Button } from "react-bootstrap";

export default Button;
```

Reply:
can i change it like example because i can set icon in button component an icon is customize prop

Reply:
yes change it like the example